### PR TITLE
fix root route

### LINKED ISSUE #93: Root route implementation
The root route (`/`) should work this way in production environment: https://github.com/meilisearch/MeiliSearch/issues/1235#issuecomment-799431012

### PR DESCRIPTION
**Pull request by irevoire**
closes #93

### CODE REVIEW

**Review by curquiza — APPROVED** [review]
LGTM 👌 
But as usual let's wait for the Marin's approval

**Review by MarinPostma — APPROVED** [review]
all good :)

bors merge

**Pull request merged**
